spyder/utils/tests/test_programs.py

```diff
@@ -282,5 +283,10 @@ def test_open_files_with_application(tmp_path):
         return_codes = open_files_with_application('not-valid.ext', fnames)
 
 
+def test_get_package_version():
+    # Primarily a test of pkg_resources being installed properly
+    assert get_package_version('IPython')
```

Comment:
Please check instead with `pyls_black`, which doesn't have a `__version__` attribute and probably won't have one in the future.

Reply:
OK! This tests get_package_version and not get_module_version, so it do test the expected thing. But it can indeed make sense to test get_module_version like that as well.

Reply:
Oh, I see. Please please leave this test as it is and add another for `get_module_version('pyls_black')`.